extensions/sql-migration/src/service/contracts.ts

```diff
@@ -548,3 +554,25 @@ export interface TdeMigrateProgressParams {
 	success: boolean;
 	message: string;
 }
+
+export interface SchemaMigrationParams {
+	sourceConnectionString: string;
+	sourceDatabaseName: string;
+	targetConnectionString: string;
+	targetDatabaseName: string;
+}
+
+export interface SchemaMigrationResult {
+	sourceDatabaseName: string;
+	targetDatabasename: string;
+	status: string;
```

Comment:
for POC string status is fine, but think we should be able to cast it directly to SchemaMigrationState here